robottelo/ui/computeresource.py

```diff
@@ -143,9 +143,125 @@ def update(self, name, newname=None, parameter_list=None,
 
     def delete(self, name, really=True):
         """Removes the compute resource entity"""
+        Navigator(self.browser).go_to_compute_resources()
         self.delete_entity(
             name,
             really,
             locators['resource.delete'],
             drop_locator=locators['resource.dropdown'],
         )
+
+    def go_to_compute_resource(self, res_name):
```

Comment:
We already have `go_to_compute_resource()` in `Navigator` class. It seems that you do more actions here than just navigation to entity page, so please provide proper name for that method